.github/workflows/compatibility.yml

```diff
@@ -12,13 +12,13 @@ jobs:
       max-parallel: 10
       matrix:
         # Minimum support version, ..., Latest version, `beta`.
-        woocommerce: [ '6.2.2', '6.3.1', '6.4.1', '6.5.1', '6.6.1', '6.7.0', 'latest', 'beta' ]
+        woocommerce: [ '6.4.1', '6.5.1', '6.6.1', '6.7.0', 'latest', 'beta' ]
         wordpress:   [ 'latest' ]
         gutenberg:   [ 'latest' ]
         php:         [ '7.4' ]
         include:
           # Edge case: oldest dependencies compatibility
-          - woocommerce: '6.2.2'
+          - woocommerce: '6.4.1'
             wordpress:   '5.8'
             gutenberg:   '11.4.0' # The latest version supporting WP 5.6.
```

Comment:
@brianyu28 do you know if we should update the Gutenberg version to the latest one supporting our oldest compatible WP version? In this case, looks like the latest version supporting 5.8 is [13.6.0](https://plugins.trac.wordpress.org/browser/gutenberg/tags/13.6.0/gutenberg.php).

Reply:
It looks like the [reason](https://github.com/Automattic/woocommerce-payments/pull/3597) for specifying the `gutenberg` version here was to avoid installing a version of Gutenberg that required a more recent WP version, and before that we would always use the latest version. So updating this to the 13.6.0, the latest version that we can, makes sense to me.

Reply:
Nice. I'll update the `gutenberg` version here when updating the workflows in a separate PR. Thanks!